browser/translate/brave_translate_browsertest.cc

```diff
@@ -49,10 +48,12 @@ namespace translate {
 
 namespace {
 const char kTestScript[] = R"(
+var api_key = undefined;
 var google = {};
 google.translate = (function() {
   return {
-    TranslateService: function() {
+    TranslateService: function(params) {
+      api_key = params.key;
```

Comment:
build-in browser js passes API key here, then it's used by go-translate scripts to make XHRs.
We emulate this behavior in tests.